src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.form

```diff
@@ -31,7 +31,7 @@
           <gridbag weightx="0.0" weighty="0.0"/>
         </constraints>
         <properties>
-          <text value="Start date (inclusive)"/>
+          <text value="  Start date (inclusive)"/>
```

Comment:
Why these spaces? If it's to "hack" text alignment, maybe you can try to align the text to the right instead? You may need to assign the text some fixed witdh for that to work.

Reply:
Noted. I will remove the spaces